db/merge_helper.h

```diff
@@ -80,7 +82,8 @@ class MergeHelper {
   Status MergeUntil(InternalIterator* iter,
                     CompactionRangeDelAggregator* range_del_agg = nullptr,
                     const SequenceNumber stop_before = 0,
-                    const bool at_bottom = false);
+                    const bool at_bottom = false,
+                    const bool allow_data_in_errors = true);
```

Comment:
Maybe I missed something. It seems the default elsewhere is false?

Reply:
Fixed